src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -90,10 +118,26 @@ public int countImages() {
     return images.size();
   }
 
+  public int countHeaders() {
+    return headers.size();
```

Comment:
Well, there's nothing more loaded after components for what can this wait. I don't have an idea how to wait for all headers.